doc/nrf/known_issues.rst

```diff
@@ -157,6 +157,16 @@ Other issues
 
 .. rst-class:: v2-0-0
 
+NCSDK-15909: TF-M may fail to build because of flash overflow with Zephyr SDK 0.14.2
```

Comment:
Hmm, this is under "Other issues", maybe place it under trusted-firmware-m instead?

Reply:
Sure, that makes sense. I did this.